public/index.html

```diff
@@ -6,22 +6,22 @@
 
   <title>Sailbot Dashboard</title>
   <meta name="description" content="Recieves and Displays data from WPI Sailbot">
-  <meta name="author" content="Sailbot Team 20-21">
+  <meta name="author" content="Sailbot Team 20-21 & Sailbot Team 21-22">
 
   <link rel="stylesheet" href="css/dashboard.css">
   <link rel="stylesheet" href="css/simple-grid.min.css">
   <link rel="stylesheet" href="css/foundation.css">
-  <link rel="shortcut icon" href="/assets/favicon.ico" type="image/x-icon">
-  <link rel="icon" href="/assets/favicon.ico" type="image/x-icon">
+  <link rel="shortcut icon" href="../public/assets/favicon.ico" type="image/x-icon">
```

Comment:
For a relative path you should be able to do the following where applicable:
```suggestion
  <link rel="shortcut icon" href="assets/favicon.ico" type="image/x-icon">
```

Reply:
Using this in new commit